src/modules/nameSchema/docs/de_DE.md

```diff
@@ -0,0 +1,65 @@
+# Namens-Schemata
+
+## Variablen
+
+| Variable              | Beschreibung                                                                         | Verwendbar in Templates für |
+|-----------------------|--------------------------------------------------------------------------------------|-----------------------------|
+| `{{unitId}}`          | ID der Einheit, z.B. `78020362`                                                      | Einheiten                   |
+| `{{unitType}}`        | Typ der Einheit (z.B. `LF 20/01`).                                                   | Einheiten                   |
+| `{{unitTypeCustom}}`  | Benutzerdefinierte Fahrzeugklasse.                                                   | Einheiten                   |
+| `{{unitAlias}}`       | Alias-Bezeichnung der Einheit (vgl. Aliase), ansonsten identisch zu `unitType`.      | Einheiten                   |
+| `{{unitIndex}}`       | Aufsteigende Nummerierung des Fahrzeugs. Parametrisierung möglich.                   | Einheiten                   |
+| `{{buildingId}}`      | ID des Gebäudes, z.B. `11221784`                                                     | Einheiten, Gebäude          |
+| `{{buildingType}}`    | Typ  der Einheit (z.B. `Feuerwehrschule`)                                            | Einheiten, Gebäude          |
```

Comment:
`der Einheit` → `des Gebäudes` plus there accidentally is an extra space after `Typ`.

Also I would prefer the `nbsp;` included in `z.B.` (change to `z.&nbsp;B.`) as required in german language: https://de.wiktionary.org/wiki/z._B.

Reply:
Ehem ;) https://github.com/search?q=repo%3ALSS-Manager%2FLSSM-V.4%20%22z.B.%22&type=code

Reply:
I do not want to hear anything :joy: :stuck_out_tongue_closed_eyes: 
Good point, I will change those later :wink: